src/cli/find-cache-file.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+const os = require("os");
+const path = require("path");
+const findCacheDir = require("find-cache-dir");
+
+/**
+ * Find default cache file (`./node_modules/.cache/prettier/.prettier-cache`) using https://github.com/avajs/find-cache-dir
+ */
+function findDefaultCacheFile() {
+  const cacheDir =
+    findCacheDir({ name: "prettier", create: true }) || os.tmpdir();
+  const cacheFilePath = path.join(cacheDir, ".prettier-cache");
+  return cacheFilePath;
+}
+
+/**
+ * @returns {string}
+ */
+function findCacheFile() {
+  return findDefaultCacheFile();
+}
```

Comment:
Rename `findDefaultCacheFile` or just export it directly?

Reply:
[2b39122](https://github.com/prettier/prettier/pull/12800/commits/2b39122e2705da9aef5bba0e1f3301a595c74830)